CONTRIBUTION.md

```diff
@@ -0,0 +1,235 @@
+ 
+2. Clone your forked repository to your local machine:
+   ```bash
+   git clone https://github.com/ForgeRock/forgerock-android-sdk.git
+   ```
+
+3. Navigate into the project folder:
+   ```bash
+    cd forgerock-android-sdk
+   ```
+   
+4. Build the project to make sure it works:
+
+   ```bash
+    ./gradlew clean build
+   ```
+
+### Understanding the Project Structure
+
+The Ping SDK for Android is organized in a modular way. This section is designed to help you 
+understand the layout of the project. We will break down each of the folders and what modules you 
+will find there. Getting familiar with the project structure will make contributing easier and 
+more efficient.
+
+    forgerock-android-sdk
+     |── e2e                               # Contains the sample app for end-to-end tests
+     |── forgerock-auth                    # Provides the OIDC client and integrates with Journey
+     |── forgerock-auth-ui*                # Contains UI components for rapid prototype apps with Journey
+     |── forgerock-authenticator           # Alloes to add Push and OATH mechanisms to the app
+     |── forgerock-core                    # Provide common functions for all modules
+     |── forgerock-integration-tests       # Includes tests for the SDK
+     |── ping-protect                      # Provide access to the Ping Protect API
+     ├── ...
+     └── ...
+
+***Note***: * Module deprecated
+
+* **e2e** : This folder houses the sample application used for end-to-end (E2E) testing of the SDK. E2E tests 
+verify that the entire system works as expected from the user's perspective, covering interactions 
+across multiple SDK components. This sample application provides a practical example of how to 
+integrate and utilize the SDK's various functionalities in a real-world scenario.
+
+* **forgerock-auth**: The forgerock-auth module is at the core of the SDK's authentication capabilities. It provides a 
+robust OpenID Connect (OIDC) client implementation, enabling seamless user authentication and 
+authorization. This module also integrates with ForgeRock's Journey framework, allowing developers
+to implement complex authentication flows and user journeys. Some features such as WebAuthn, 
+Social Login and many others are also included in this module.
+
+* **forgerock-auth-ui**: The forgerock-auth-ui module provides a collection of pre-built UI components designed to expedite 
+the development of prototype applications. These UI elements integrate directly with the 
+forgerock-auth module and are specifically designed to facilitate rapid prototyping with the 
+Journey framework. This module is deprecated.
+
+* **forgerock-authenticator**: This module empowers you to add powerful multi-factor authentication (MFA) capabilities to an 
+application. It includes support for Push notifications and Time-based One-Time Passwords (TOTP) 
+mechanisms (OATH), enabling developers to enhance security and protect users' accounts.
```

Comment:
We're mixing second- and third-person a bit in this paragraph.
Either:
 - Switch "developers" to "you"
 Or
 - Switch "empowers you to" to "lets developers"
 
 Based on the text so far, second-person "you" is probably preferred.